test/buy-tshirt.spec.ts

```diff
@@ -16,14 +16,9 @@ describe('Buy a t-shirt', () => {
   const bankPaymentPage: BankPaymentPage = new BankPaymentPage();
   const orderSummaryPage: OrderSummaryPage = new OrderSummaryPage();
 
-  beforeEach(() => {
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 210000;
-  });
-
   it('then should be bought a t-shirt', async () => {
     await browser.waitForAngularEnabled(false);
-    await browser.get('http://automationpractice.com/');
-    await (browser.sleep(6000));
+    await browser.get('http://automationpractice.com/index.php?id_category=1');
```

Comment:
What is the reason of adding the /index.php?id_category=1 to the route?

Reply:
The home page is not working, categories are not showing up